### PR TITLE
Update get started intro section

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Update 'Get Started with Realm' section of Introduction. Removed outdated references to Task Tracker tutorial. Just redirected users to each SDK's landing page b/c now they've been filled with all kinds of useful introduction/getting started tasks.

### Jira

n/a

### Staged Changes (Requires MongoDB Corp SSO)

- [Introduction](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/intro_tweak/introduction)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/introduction.txt [hunk 1]
```diff
@@ -112,27 +112,17 @@ with data from your app:
...
-
-- :doc:`Flutter SDK </sdk/flutter>` for Flutter
-- :doc:`Java SDK </sdk/java>` for Java
-- :doc:`Kotlin SDK </sdk/kotlin>` for Kotlin
-- :doc:`.NET SDK </sdk/dotnet>` for C# and Xamarin
-- :doc:`Node.js SDK </sdk/node>` for Node.js
-- :doc:`React Native SDK </sdk/react-native>` for TypeScript and JavaScript with React Native
-- :doc:`Swift SDK </sdk/swift>` for Swift and Objective-C
-- :doc:`Web SDK </web>` for browser JavaScript
+Learn more about the Realm SDKs in their documentation:
```
Comment (krollins-mdb) [inline source/introduction.txt:115]: "in their documentation" feels like these links may go somewhere else entirely (like another website). I suggest rewording a bit.
```suggestion
You can use Realm with many different languages through our SDKs. Each Realm SDK page has more information about how to get started.
```

Comment (mongodben) [reply source/introduction.txt:115]: how bout we get rid of "in their documentation" to just:

```suggestion
Learn more about the Realm SDKs:
```

**Review by krollins-mdb — APPROVED** [review]
Ship it!
